discovery-provider/src/tasks/index_plays.py

```diff
@@ -33,8 +33,7 @@ def get_track_plays(self, db):
         most_recent_play_date = session.query(
             Play.updated_at
         ).order_by(
-            desc(Play.updated_at),
-            desc(Play.id)
+            desc(Play.updated_at)
```

Comment:
did we have the secondary sort here for a reason?

Reply:
I wrote this the first time I believe and can't think of a reason why it would need a secondary sort. It's just getting the most recent date. 
Actually I think this query was pretty slow on prod so a couple weeks ago we put an index on play.id and play.updated_at together. But, that index not in our migrations. There is also an index in the migrations for just play.updated_at which should keep this query fast.

Reply:
what if two have the same updated_at? won't this be non-deterministic then?

Reply:
@jowlee does having the secondary sort affect performance? if not, we should just not touch ii

Reply:
It didn't seem to, but I believe it's because of the index on both cols that isn't in the migrations. But can just leave as it.